go/ethadapter/geth_rpc_client.go

```diff
@@ -143,7 +143,7 @@ func (e *gethRPCClient) Nonce(account gethcommon.Address) (uint64, error) {
 }
 
 func (e *gethRPCClient) BlockListener() (chan *types.Header, ethereum.Subscription) {
-	ch := make(chan *types.Header, 1)
+	ch := make(chan *types.Header, 100)
```

Comment:
A comment on this might be nice to say what the channel is used for (source and destination) and why it has a buffer.

Thinking along the lines of `// this channel holds blocks that have been received from the geth network but not yet processed by the enclave, the buffer provides resilience in case of intermittent RPC or processing issues`

Reply:
That's great 👌